src/app/toolbar/toolbar.component.html

```diff
@@ -3,7 +3,8 @@
     <form class="toolbar-pf-actions" [ngClass]="{'no-filter-results': !filterFields}" (submit)="submit($event)">
       <div class="form-group toolbar-apf-filter">
         <alm-filter-fields [config]="config.filterConfig" *ngIf="config.filterConfig && config.filterConfig.fields"
-                           (onAdd)="addFilter($event)"></alm-filter-fields>
+                           (onAdd)="addFilter($event)"
+                           (onSelectType)="filterTypeSelect($event)"></alm-filter-fields>
```

Comment:
I had envisioned this as an onKeyPress event that developers would listen for. The developer would perform the actual filter of available list options and adjust the filter config appropriately. Again, each filter field has its own set of unique properties, options, etc.